### PR TITLE
[Tests-only] Add acceptance test to delete local storage using occ command

### LINKED ISSUE #36704: Add CLI acceptance tests for files_external occ commands
We have some UI tests for adding local storage in the admin storage settings. But we have very little tests for the files_external occ commands that create/modify/delete storages.

Add test coverage for:

- [x] files_external:applicable     #37012      #36707   
- [x] files_external:backends            #37035   
- [x] files_external:config            #37015     
- [x] files_external:create         #36707        #36689
- [x] files_external:delete         #36756        
- [x] files_external:export       #36794          
- [x] files_external:import #37513 
- [x] files_external:list             #36754             
- [x] files_external:option            #37018     
- [x] files_external:verify       #36806

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance test to delete local storage using occ command

## Related Issue

- Related #36704 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -364,6 +369,16 @@ public function deleteLastBackgroundJobUsingTheOccCommand($job) {
...
 	}
 
+	/**
+	 * List created local storage mount
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function listLocalStorageMount() {
+		\OccContext::invokingTheCommand('files_external:list --output=json');
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
		$this->invokingTheCommand('files_external:list --output=json');
```
when you are inside occ context its better to use `this`

File: tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  As an admin
+  I want to delete local storage
+  So that I can remove undesired local storage
+
+  Scenario: delete local storage
+    Given the administrator has created the local storage mount "local_storage2"
+    And the administrator has created the local storage mount "new_local_storage"
+    And the administrator has uploaded file with content "file in local storage" to "/local_storage2/file-in-local-storage.txt"
+    And the administrator has uploaded file with content "new file" to "/new_local_storage/new-file"
+    When the administrator deletes folder "local_storage2" using occ command
```
Comment (dpakach) [inline tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature]: Currently even when the occ command fails for some reason the test will pass. Need to do something about that.

File: tests/acceptance/features/cliLocalStorage/listLocalStorage.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  I want to list all created local storage from the command line
+  So that I can view available local storage
+
+  Scenario: List the created local storage
+    Given the administrator has created the local storage mount "local_storage2"
+    And the administrator has created the local storage mount "new_local_storage"
+    And the administrator has uploaded file with content "this is a file in local storage" to "/local_storage2/file-in-local-storage.txt"
+    And the administrator has uploaded file with content "new file" to "/new_local_storage/new-file"
+    When the administrator lists all the created local storage using occ command
+    Then the following local storage should exist
```
Comment (dpakach) [inline tests/acceptance/features/cliLocalStorage/listLocalStorage.feature]:
```suggestion
    Then the local storage returned by the occ command should include
```

File: tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (jasson99) [reply tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature]: fixed

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 4]
```diff
@@ -1204,6 +1204,53 @@ public function theFollowingLocalStoragesShouldExist(TableNode $mountPoints) {
...
+		foreach ($commandOutput as $i) {
+			$createdLocalStorage[$i->mount_id] = \ltrim($i->mount_point, '/');
+		}
+		foreach ($expectedLocalStorages as $i) {
+			Assert::assertNotContains($i['localStorage'], $createdLocalStorage);
+		}
+	}
+
+	/**
+	 * @When the administrator deletes local storage :folder using occ command
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
	 * @When the administrator deletes local storage :folder using the occ command
```

File: tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature [hunk 5]
```diff
@@ -0,0 +1,19 @@
+@cli @skipOnLDAP @local_storage
+Feature: create local storage from the command line
```
Comment (phil-davis) [inline tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature]:
```suggestion
Feature: delete local storage from the command line
```

**Review by phil-davis — COMMENTED** [review]
and please rebase - that should remove the old createLocalStorage commits from the list in this PR

File: tests/acceptance/features/cliLocalStorage/listLocalStorage.feature (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/features/cliLocalStorage/listLocalStorage.feature]: This is actually from another pr.

File: tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature (hunk 5 above)
Comment (jasson99) [reply tests/acceptance/features/cliLocalStorage/deleteLocalStorage.feature]: fixed

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 4 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/OccContext.php]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**